Runtime/Components/CinemachineSplineDolly.cs

```diff
@@ -0,0 +1,399 @@
+#if CINEMACHINE_UNITY_SPLINES
+using UnityEngine;
+using System;
+using Cinemachine.Utility;
+using UnityEngine.Splines;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// A Cinemachine Virtual Camera Body component that constrains camera motion
+    /// to a Spline.  The camera can move along the path.
+    ///
+    /// This behaviour can operate in two modes: manual positioning, and Auto-Dolly positioning.
+    /// In Manual mode, the camera's position is specified by animating the Path Position field.
+    /// In Auto-Dolly mode, the Path Position field is animated automatically every frame by finding
+    /// the position on the path that's closest to the virtual camera's Follow target.
+    /// </summary>
+    [DocumentationSorting(DocumentationSortingAttribute.Level.UserRef)]
```

Comment:
This is an obsolete attribute.  Don't use it in new classes.  Remove everywhere in this file.